melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/sed/mapper/til_sed/SedMapper.java

```diff
@@ -75,17 +76,24 @@ default Person hentPerson(SedDataDto sedData) {
         person.setFoedselsdato(formaterDato(sedData.getBruker().getFoedseldato()));
         person.setFoedested(null); //det antas at ikke trengs når NAV fyller ut.
         person.setKjoenn(Kjønn.valueOf(sedData.getBruker().getKjoenn()));
-
-        var statsborgerskap = new Statsborgerskap();
-        statsborgerskap.setLand(LandkodeMapper.getLandkodeIso2(sedData.getBruker().getStatsborgerskap()));
-
-        person.setStatsborgerskap(Collections.singletonList(statsborgerskap));
-
+        person.setStatsborgerskap(hentStatsborgerskap(sedData));
         person.setPin(hentPin(sedData));
 
         return person;
     }
 
+    default List<Statsborgerskap> hentStatsborgerskap(SedDataDto sedDataDto) {
+        return sedDataDto.getBruker().getStatsborgerskap().stream()
+            .map(this::lagStatsborgerskap)
+            .collect(Collectors.toList());
+    }
+
+    private Statsborgerskap lagStatsborgerskap(String landkode) {
+        Statsborgerskap statsborgerskap = new Statsborgerskap();
+        statsborgerskap.setLand(LandkodeMapper.getLandkodeIso2(landkode));
```

Comment:
Vet du om vi alltid sender inn ISO2 eller 3? 🤔  Lurer på om ikke denne konverteringen kunne vært fjernet her.

Reply:
Usikker på hvorfor det er det sjæl, men jeg så at alle andre steder i koden så konverteres det mot IS02, men i melosys-api er det ISO3. Jeg tenkte at kanskje det er noen formattering som er et krav på en SED?

Reply:
Klassen `Land`, `Statsborgerskap` fra TPS + PDL og dermed inngangsvilkår bruker ISO-3. Fag og saksbehandlere ønsker ellers å forholde seg til ISO-2 hvis det skal vises. Vi kunne konvertere i melosys-api for å slippe å konvertere her.